#! /bin/sh /usr/share/dpatch/dpatch-run
## mp3tag_string_termination patch by arno <arenevier@fdn.fr>
## http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=358929 

--- cdcat-1.01b/src/mp3tag.cpp	2005-08-22 19:03:00.000000000 +0200
+++ cdcat-1.01b~/src/mp3tag.cpp	2006-08-13 15:59:20.000000000 +0200
@@ -219,7 +219,7 @@
   unsigned int pos=0;
   unsigned int framelen;
   char id[5];
-  id[5] ='\0';
+  id[4] ='\0';
 
   fseek(mp3,0,SEEK_END);
   flen = ftell(mp3);
